Documentation/README.md

```diff
@@ -1,20 +1,99 @@
 # Resources
-
 See the following resources to get started.
 
-## .NET Core
+## System.Device.* APIs
 
-* [.NET Core documentation](https://docs.microsoft.com/dotnet/)
-* [Install .NET Core on Raspberry Pi](https://github.com/dotnet/core/blob/master/samples/RaspberryPiInstructions.md)
+* [Device Bindings](https://github.com/dotnet/iot/tree/master/src/devices) - Includes a collection of APIs representing a range of sensors, displays and human interface devices based on System.Device.* APIs.
+* [DevicesApiTester CLI](https://github.com/dotnet/iot/tree/master/tools/DevicesApiTester) - Helpful utility, based on System.Device.* APIs, that include various commands for testing connected development boards and external hardware.
+
+## .NET Core
+* [.NET Core Documentation](https://docs.microsoft.com/dotnet/)
 * [.NET Core ARM64 Status](https://github.com/dotnet/announcements/issues/82)
 * [.NET Core Docker Samples](https://github.com/dotnet/dotnet-docker/tree/master/samples)
 
-## Raspberry Pi
+## Interface Knowledge Base
+### General-Purpose Input/Output (GPIO)
+* [GPIO Wiki](https://en.wikipedia.org/wiki/General-purpose_input/output)
+* [Digital I/O Fundamentals](http://www.ni.com/white-paper/3405/en/#toc1)
+
+### Inter-Integrated Circuit (I2C)
+* [I2C Wiki](https://en.wikipedia.org/wiki/I%C2%B2C)
+* [I2C Tutorial](https://learn.sparkfun.com/tutorials/i2c/all)
+
+### Serial Peripheral Interface (SPI)
+* [SPI Wiki](https://en.wikipedia.org/wiki/Serial_Peripheral_Interface)
+* [SPI Tutorial](https://learn.sparkfun.com/tutorials/serial-peripheral-interface-spi/all)
+
+## Development Boards
```

Comment:
I know you probably went with lexicographically ordering the boards but I think that it makes more sense to have the RPi first and then the HummingBoard since those are the two devices where we've done most of our testing. Even though our Generic driver should work on BeagleBoard and Pine64, we haven't done extensive testing there so I would make it clear here that our two main supported boards are RPi and HB (for preview 1 at least, As our library matures and once we add automated tests running on boards then we can start adding more boards to the supported list.). Anyway, I'm fine if you want to keep the boards listed, but we should make it clear that if you want to try our library, RPi and HB are the way to go for now.